gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/entities.proto

```diff
@@ -44,6 +44,7 @@ message EntitiesRequest {
 
   int32 limit = 20;
   int32 offset = 21;
+  bool query_live_entities_only = 22;
```

Comment:
The flag is not backward compatible. Because, it would be `false` by default, and we want to query live entities only by default to make it backward compatible.

Looking for better names for the flag, so that it can be reverted and can stay in the backward compatible mode